File: pythatriple-web-api/src/main/java/com/example/pythatriple_web_api/repository/TripleResultRepository.java
package com.example.pythatriple_web_api.repository;

import java.util.Optional;

import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

import com.example.pythatriple_web_api.model.TripleResult;

@Repository
public interface TripleResultRepository extends JpaRepository<TripleResult, Long> {
	Optional<TripleResult> findByAAndBAndC(Integer a, Integer b, Integer c);
}

File: pythatriple-web-api/src/main/kotlin/com/example/pythatriple_web_api/repository/TripleResultRepository.kt
package com.example.pythatriple_web_api.repository

import com.example.pythatriple_web_api.model.TripleResult
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface TripleResultRepository : JpaRepository<TripleResult, Long> {
	fun findByAAndBAndC(a: Int, b: Int, c: Int): TripleResult?
}
